src/test/integration/fastapi/tests/app_runner.py

```diff
@@ -2,17 +2,18 @@
 import subprocess
 import sys
 from pathlib import Path
-from test.test_utils.processes import kill_process
+from test.test_utils.processes import kill_process, wait_for_process_output
 
 
 class FastApiApp(object):
-    def __init__(self, app: str, port: int):
+    def __init__(self, app: str, port: int, env: dict = {}):
```

Comment:
this is generally bad to put here mutable variables. Make it nullable?

Reply:
i've made it an `Optional[dict]`, i hope using `**(env or {})` is reasonable